File: luci-app-iptv-manager/luasrc/model/cbi/iptv-manager/epg.lua
local m, s, o

m = Map("iptv", translate("Настройки телепрограммы (EPG)"),
    translate("Загрузка и управление EPG. EPG хранится в RAM (/tmp) и не занимает флеш-память."))

m:chain("iptv")

o = m:option(Value, "epg_url", translate("URL EPG (XMLTV)"))
o.placeholder = "http://epg.cdntv.online/lite.xml.gz"
o.datatype = "string"
o.description = "Поддерживаются XML и XML.gz. Рекомендуемый: http://epg.cdntv.online/lite.xml.gz"

return m

File: luci-app-iptv-manager/htdocs/luci-static/resources/view/iptv-manager/epg.js
'use strict';
'require view';
'require form';
'require uci';

return view.extend({
    load: function() {
        return Promise.all([uci.load('iptv')]);
    },

    render: function() {
        var m, s, o;

        m = new form.Map('iptv', _('Телепрограмма (EPG)'),
            _('Загрузка и управление EPG. EPG хранится в RAM (/tmp) и не занимает флеш-память.'));

        s = m.section(form.TypedSection, 'main', null, true);
        s.anonymous = true;

        o = s.option(form.Value, 'epg_url', _('URL EPG (XMLTV)'));
        o.placeholder = 'http://epg.cdntv.online/lite.xml.gz';
        o.description = _('Поддерживаются XML и XML.gz. Рекомендуемый: http://epg.cdntv.online/lite.xml.gz');

        return m.render();
    }
});
